KARL_CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased]
 
+- Feature: Update `font` on `body` class with Kitten V2.
```

Comment:
with kitten V2 -> in kitten V2?

Reply:
C'est avec la mise à jour de `Kitten`. Si on met `in Kitten V2` oui ça marche aussi mais je préfère `with kitten V2` et je l'ai écrit partout en plus.

Reply:
En même temps il va sans doute falloir repasser partout si on ne parle pas de Kitten v2 mais de Style guide v2 https://github.com/KissKissBankBank/kitten/pull/853